src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -498,6 +504,11 @@ public Response get(@Context HttpHeaders httpHeaders, String json) throws Except
 			span.setTag("query_timeout", true);
 			span.log(e.getMessage());
 			return setHeaders(Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity(new ErrorResponse(e.getMessage()))).build();
+		} catch (QueryRejectedException e) {
+			logger.error("Query was rejected.", e);
+			Tags.ERROR.set(span, Boolean.TRUE);
```

Comment:
do we really want to tag it as error in lightstep? lightstep will light up as a christmas light if we enable query rejection, and this will hide any actual errors that we would want to see.

Reply:
good point, thx